demos/python_demos/image_inpainting_demo/image_inpainting_demo.py

```diff
@@ -33,30 +34,103 @@ def build_argparser():
                       help="Optional. Specify the target device to infer on; CPU, GPU, FPGA, HDDL or MYRIAD is "
                            "acceptable. The demo will look for a suitable plugin for device specified. "
                            "Default value is CPU", default="CPU", type=str)
-    args.add_argument("-p", "--parts", help="Optional. Number of parts to draw mask.", default=8, type=int)
-    args.add_argument("-mbw", "--max_brush_width", help="Optional. Max width of brush to draw mask.", default=24, type=int)
-    args.add_argument("-ml", "--max_length", help="Optional. Max strokes length to draw mask.", default=100, type=int)
-    args.add_argument("-mv", "--max_vertex", help="Optional. Max number of vertex to draw mask.", default=20, type=int)
-    args.add_argument("--no_show", help="Optional. Don't show output", action='store_true')
+    args.add_argument("-r", "--rnd",
+                      help="Optional. Use random mask for inpainting (with parameters set by -p, -mbw, -mk and -mv).Skipped in GUI mode",
+                      action='store_true')
+    args.add_argument("-p", "--parts", help="Optional. Number of parts to draw mask. Skipped in GUI mode", default=8, type=int)
+    args.add_argument("-mbw", "--max_brush_width", help="Optional. Max width of brush to draw mask. Skipped in GUI mode",
+                      default=24, type=int)
+    args.add_argument("-ml", "--max_length", help="Optional. Max strokes length to draw mask. Skipped in GUI mode",
+                      default=100, type=int)
+    args.add_argument("-mv", "--max_vertex", help="Optional. Max number of vertex to draw mask. Skipped in GUI mode",
+                      default=20, type=int)
+    args.add_argument("-mc", "--mask_color",
+                      help="Optional. Color to be treated as mask (provide 3 RGB components in range of 0...255)."
+                      " Default is 0 0 0. Skipped in GUI mode", default=[0, 0, 0], type=int, nargs="+")
+    args.add_argument("--no_show", help="Optional. Don't show output. Skipped in GUI mode", action='store_true')
```

Comment:
Why error ? It is ignored - that's sufficient for normal operation

Reply:
A user who specifies `--no_show` will expect that the GUI will not be shown; if this is impossible, then the application should notify the user instead of silently ignoring the option.

Reply:
> let's keep these parameters as optional

Applying @IRDonch's suggestion keeps `--no_show` optional.

> It's noted in help that they will be ignored in GUI mode. I don't want to confuse users with extra error messages indeed.

That requires careful help reading. Error notification is more explicit and less confusing as it detects a case containing two opposite settings: gui mode and disabling that gui.

Reply:
done

Reply:
OK, but you need to update the parameter description to reflect the new semantics.

Reply:
Parameter description is updated, but it is not shown here for some GitHub reason. Now it says "Cannot be used in GUI mode"